docs/apis/batch/libs/table.md

```diff
@@ -238,6 +238,17 @@ Table result = left.union(right);
       </td>
     </tr>
 
+    <tr>
+      <td><strong>Distinct</strong></td>
+      <td>
+        <p>Similar to a SQL DISTINCT clause. Removes duplicate values and returns only distinct (different) values.</p>
```

Comment:
We should make clear that rows with distinct value combinations are returned.
How about: "Returns rows with distinct value combinations."